code/src/pages/cart/CartPage.tsx

```diff
@@ -0,0 +1,127 @@
+import { useState } from "react";
+
+import AppBox from "@/components/app-box/AppBox";
+import AppButton from "@/components/app-button/AppButton";
+import AppTypography from "@/components/app-typography/AppTypography";
+
+import { items } from "@/pages/cart/CartPage.constants";
+import CartItem from "@/pages/cart/components/CartItem";
+import formatPrice from "@/utils/format-price/formatPrice";
+
+import "@/pages/cart/CartPage.scss";
+
+type CartItemsType = {
+  image: string;
+  name: string;
+  productPrice: number;
+  quantity: number;
+  calculatedPrice: number;
+}[];
```

Comment:
In this situations it is better to create `CartItem` type and then use `CartItem[]`. It is considered more readable

Reply:
> In this situations it is better to create `CartItem` type and then use `CartItem[]`. It is considered more readable

moved `[]` to `useState` like `useState<CartItemsType[]>`